webpack.config.js

```diff
@@ -44,7 +44,7 @@ if (production) {
 module.exports = [
   {
     context: __dirname,
-    devtool: production ? undefined : 'eval-source-map',
+    devtool: production ? undefined : 'eval',
```

Comment:
This means we won't be emitting sourcemaps at all, but the desire is to emit sourcemaps, but only in development mode.

Is it possible for us to emit sourcemaps without seriously slowing down execution locally? Actually, it is a bit weird that just having a source map makes execution of the code slow. Any theories on why that's causing this?